packages/frontend/src/components/Drawer/index.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react'
```

Comment:
any chance we can get rid of this custom Drawer component completely?

Reply:
i think a bit tricky cus chakra dont have mini variant drawers like MUI does: https://mui.com/material-ui/react-drawer/#mini-variant-drawer, so if we want to, need some time to reinvent it